lua/nvim-tree/renderer/builder.lua

```diff
@@ -294,14 +294,17 @@ function Builder:build(tree)
   return self
 end
 
-local function format_root_name(root_cwd, modifier)
-  local base_root = utils.path_remove_trailing(vim.fn.fnamemodify(root_cwd, modifier))
-  return utils.path_join { base_root, ".." }
```

Comment:
We have lost the `/..` at the end of the default which might upset some users.

Proposal: append it to the default: `root_folder_label = ":~:s?$?/..?"`

That adds some complexity. I'd appreciate some test/feedback.

Reply:
Doesn't matter how complex default is. Most users won't ever change it.